Sources/SpeziContact/Models/ContactOption.swift

```diff
@@ -11,6 +11,18 @@ import SwiftUI
 
 
 /// Customizable way to get in contact with an individual and usually connected to a ``Contact``.
+///
+/// The following example demonstrates the usage of the `Contact`.
```

Comment:
`Contact` is not correct here, `ContactOption` (referenced via backticks) would be more suitable